pkg/controller/machineset/controller.go

```diff
@@ -26,7 +27,6 @@ import (
 
 	machinev1beta1 "github.com/openshift/machine-api-operator/pkg/apis/machine/v1beta1"
 	"github.com/openshift/machine-api-operator/pkg/util"
```

Comment:
Seems the machineSet changes are for consistency but not really related to the BZ referenced, they probably deserve its own commit?

Reply:
I've split the commit into two, one for vSphere and one for MachineSet